css/master.css

```diff
@@ -6,6 +6,14 @@
   --section-main-padding: 50px;
 }
 
+html {
+  scroll-behavior: smooth;
+}
+
```

Comment:
- [OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `over-flow: hidden;` to do that, which will make your mobile menu correspond to the original design.